src/dbnode/storage/bootstrap/bootstrapper/fs/source.go

```diff
@@ -706,6 +709,7 @@ func (s *fileSystemSource) read(
 	}
 
 	if run == bootstrapIndexRunType {
+		span.LogEvent("bootstrap_from_index_persisted_blocks_start")
```

Comment:
Maybe add the `md.ID().String()` and `shardTimeRanges.SummaryString()` as tags to this span?